src/app/content/highlights/components/CardWrapper.tsx

```diff
@@ -50,9 +50,9 @@ const Wrapper = ({highlights, className, container, highlighter}: WrapperProps)
     if (element.current.contains(activeElement)) {
       focusedHighlight.focus();
     } else {
-      setShouldFocusCard(true);
+      setShouldFocusCard(focusedId !== undefined);
```

Comment:
I'm trying to reason about this change but I'm getting stuck... it seems you're fixing the case when `focusedId === undefined`. But would it even get to this point? `focusedHighlight` should not find anything in that case unless `highlight.id` is also undefined. Is this something that happens?

Reply:
This is something that happens due to timing issues in Firefox. In some cases, deleting a highlight followed by highlighting some text caused the focus to jump to the highlight editor, causing the color to be selected automatically. This change stopped that behavior.